src/KafkaW/ProducerTopic.cpp

```diff
@@ -69,33 +69,20 @@ int ProducerTopic::produce(unique_ptr<Producer::Msg> &Msg) {
   x = rd_kafka_produce(RdKafkaTopic, partition, msgflags, Msg->data, Msg->size,
                        key, key_len, Msg.get());
 
-  auto &s = Producer_->Stats;
+  auto &s = KafkaProducer->Stats;
   if (x != 0) {
     auto err = rd_kafka_last_error();
-    bool print_err = true;
     if (err == RD_KAFKA_RESP_ERR__QUEUE_FULL) {
       ++s.local_queue_full;
-      if (print_err) {
-        LOG(Sev::Warning, "QUEUE_FULL  outq: {}",
-            rd_kafka_outq_len(Producer_->getRdKafkaPtr()));
```

Comment:
I think these errors+warnings are important, data is being lost.

Reply:
The removed log messages are not being lost as such with this change as `print_err` was hardcoded to `false`. I am fine with putting back the calls to `LOG()` if you have a suggestion for what to do with `print_err`.

Reply:
Looks like it was set to `true`, not `false`?
I think `print_err` should be removed and simply put the `LOG`s back without the `if` statements.

Reply:
Oops, sorry about that. You are entirely correct. I will do exactly what you suggest.

Reply:
Done. I guess I was somewhat tired when I made the changes yesterday.